components/usage/pkg/stripe/stripe.go

```diff
@@ -36,19 +38,54 @@ func Authenticate(apiKeyFile string) error {
 	return nil
 }
 
-// FindCustomersForTeamIds queries the stripe API to find all customers with a teamId in `teamIds`.
-func FindCustomersForTeamIds(teamIds []string) {
+// UpdateUsage updates teams' Stripe subscriptions with usage data
+// `usageForTeam` is a map from team name to total workspace seconds used within a billing period.
```

Comment:
I think `usageForTeam` will soon become a bit more complex, due to there being multiple workspace sizes/classes, which would consume credits at different rates.

But for now it's fine to have them as just seconds without workspace class information.